tests/testthat/helper-ckanr.R

```diff
@@ -1,25 +1,72 @@
+#' Test whether the configured test CKAN is offline
+#'
+#' @keywords internal
+#' @importFrom testthat skip
+#' @details
+#' \code{check_ckan} will allow a test to fail (and not hide error messages)
+#' if \code{setup_ckanr(test_strict="TRUE")} was set.
+#' \code{check_ckan} will allow a test to skip (and the test suite to pass)
+#' if \code{setup_ckanr(test_strict)} was set to anything but the string "TRUE".
+#' @param url A URL that shall be tested whether it is online
+#'
```

Comment:
I probably should have used inline comments then. Just wanted to document this function to you and Wush - what's the best formal way to document internal functions?